tendrl/node_agent/message/handler.py

```diff
@@ -37,12 +37,21 @@ def read_socket(self, sock, *args):
             msg = struct.unpack(frmt, data)
             message = Message.from_json(msg[0])
             # Logger is in commons so passing alert from here
-            notify_only = message.payload.get('notify_only_ui', False)
-            if message.priority == NOTICE_PRIORITY and not notify_only:
```

Comment:
@shtripat since notify_only_ui is removed. Make these changes to job status alerts too in commons

Reply:
@r0h4n sent https://github.com/Tendrl/commons/pull/728 for this.
Also gluster-integration `sds_sync/event_utils.py` need changes. @GowthamShanmugam do the needful as discussed and verify the scenario volume stop and start and make sure alerts are created and cleared respectively in this case.

Reply:
@shtripat @r0h4n i am doing the changes in sds_sync/event_utils.py